src/garrysmod/Dockerfile

```diff
@@ -8,3 +8,30 @@ RUN set -ex \
     +quit
 
 COPY cfg/mount.cfg garrysmod/cfg/mount.cfg
+
+# Switch to x64 branch
+RUN set -ex \
+    && /usr/games/steamcmd +login anonymous \
+    +app_update 4020 \
+    -beta x86-64 \
+    validate \
+    +quit
+
+USER root
```

Comment:
docker-steamcmd (two levels upstream) [creates a `steam` user](https://github.com/rbreslow/docker-steamcmd/blob/3312e87dc3edd0ab22c0b67abbac56fccfe14bfb/Dockerfile#L5-L6) that is used by default. 

We have to switch back to `root` to install packages.